sdk/eventhub/event-hubs/test/stress/app/package.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "stress-test-track-2",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "tsc -p .",
+    "test": "npm run build && node dist/src/scenarioNoActivity.js --testDurationInMs=10000"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "applicationinsights": "^1.8.8",
+    "@azure/event-hubs": "5.9.0",
```

Should that be latest?
```suggestion
    "@azure/event-hubs": "5.10.0",
```